conversation/frontend/src/features/message-edit/components/RecipientListEdit/RecipientListEdit.tsx

```diff
@@ -0,0 +1,145 @@
+import { Combobox } from '@edifice.io/react';
+import { Fragment, ReactNode, useEffect, useState } from 'react';
+import { useSearchRecipients } from '~/features/message-edit/hooks/useSearchRecipients';
+import { useI18n } from '~/hooks';
+import { Group, Recipients, User } from '~/models';
+import { Visible } from '~/models/visible';
+import { useAppActions, useMessageUpdated } from '~/store';
+import { RecipientListItem } from './RecipientListItem';
+import { MessageRecipientListSelectedItem } from './RecipientListSelectedItem';
+
+export interface RecipientListProps {
+  recipients: Recipients;
+  head: ReactNode;
+  recipientType: 'to' | 'cc' | 'cci';
+  onRecipientUpdate?: (recipients: Recipients) => void;
+}
+
+export function MessageRecipientListEdit({
+  recipients,
+  head,
+  recipientType,
+  onRecipientUpdate,
+}: RecipientListProps) {
+  const { t } = useI18n();
+  const [recipientArray, setRecipientArray] = useState<(User | Group)[]>([]);
+
+  const messageUpdated = useMessageUpdated();
+  const { setMessageUpdated, setMessageUpdatedNeedToSave } = useAppActions();
+
+  const onRecipientSelected = (recipient: Visible) => {
```

Comment:
En fait, c'est un handler : 
```suggestion
  const handleRecipientClick = (recipient: Visible) => {
```